src/crl/types.rs

```diff
@@ -101,7 +103,12 @@ impl<'a> CertRevocationList<'a> {
         ) {
             // If the certificate has no CRL distribution points, and the CRL has no issuing distribution point,
             // then we can consider this CRL authoritative based on the issuer matching.
-            (cert_dps, None) => return cert_dps.is_none(),
+            (None, None) => return true,
```

Comment:
Don't love that this has duplicate arms that do the same thing. Maybe better to just do `(_, None) => return true,` and collapse the comments to explain why this makes sense?

Reply:
Can the `path.cert.crl_distribution_points()` term be removed from the match altogether? I think this is equivalent to `match self.issuing_distribution_point() { None => return true; Some(crl_idp) => <parse it> }`

Reply:
Reworked, thanks!